crates/nu-command/tests/commands/cp.rs

```diff
@@ -318,3 +318,17 @@ fn copy_dir_symlink_file_body_not_changed() {
         assert!(actual.contains("hello_data"));
     });
 }
+
```

Comment:
Can you please add a comment says that it doesn't work in windows, needs more work. ?

For the remaining code, looks good to me :-)